modules/sno-du-configuring-performance-addons.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-performance-addons_{context}"]
+= Configuring the Performance Addon Operator
+
+This is a key configuration for the single node distributed unit (DU).
+Many of the real time capabilities and service assurance is configured here.
+
+.Procedure
+
+* Configure the performance addons using the following example:
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: perfprofile-policy
+spec:
+  additionalKernelArgs:
+    - idle=poll
+    - rcupdate.rcu_normal_after_boot=0
+  cpu:
+    isolated: 2-19,22-39 <1>
+    reserved: 0-1,20-21 <2>
+  hugepages:
+    defaultHugepagesSize: 1G
+    pages:
+      - count: 32 <3>
+        size: 1G <4>
+  machineConfigPoolSelector:
+    pools.operator.machineconfiguration.openshift.io/master: ""
+  net:
+    userLevelNetworking: true <5>
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  numa:
+    topologyPolicy: restricted
+  realTimeKernel:
+    enabled: true    <6>
+----
+
+<1> Set the isolated CPUs. Ensure all of the HT pairs match.
+<2> Set the reserved CPUs.  In this case, a hyperthreaded pair is allocated on NUMA 0 and a pair on NUMA 1.
+<3> Set the huge page size.
+<4> Set the huge page number.
+<5> This setting isolates the CPUs from networking interrupts.
+<6> This installs the real-time Linux kernel.
```

Comment:
Set to true to install... (keep the bullets parallel)

Reply:
changed.